spec/javascripts/health-care/components/error-message.spec.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-addons-test-utils';
+import ErrorMessage from '../../../../_health-care/_js/_components/error-message';
+import _ from 'lodash';
+
+describe('<ErrorMessage>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <ErrorMessage message="There is an error here"/>
+    );
+    assert.ok(component, "Cannot even render component");
+  });
+
+  it('has sane looking features', () => {
+    const inputs = ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'span');
+    expect(inputs).to.have.length(1);
+  });
+
+  it('has the appropriate className', () => {
```

Comment:
Had to write a test to make sure the class is included so that my test in `SocialSecurityNumber` that checks to make sure `ErrorMessage` is included in the case of an error works.